schema/organization/Product.md

```diff
@@ -0,0 +1,36 @@
+# Product
+
+The `Product` type allows you to provide details about a product such as the product brand, logo and ID. This type can be used as any kind of product that is not [`CreativeWork`](/CreativeWork) item.
+
+## Examples
+
+The examples below are based on a model of [direct vision spectroscope](https://www.eiscolabs.com/collections/spectroscopes/products/ph0595).
+
+```json
+{
+    "type": "Product",
+    "brand": "eisco",
+    "logo": "https://beta-static.fishersci.com/content/dam/fishersci/en_US/images/brands/e/eisco/eisco-logo-1071.png",
+    "productID" : "PH0595"
+}
+```
+
+YAML provides a more readable format for providing details about a product in places like Markdown front-matter.
+
+```markdown
+---
+title: Laboratory Catalogue
+content:
+  - product:
+      - brand: eisco
+        logo: https://beta-static.fishersci.com/content/dam/fishersci/en_US/images/brands/e/eisco/eisco-logo-1071.png
+        productID: PH0595
+---
+```
+
```

Comment:
Perhaps this is over the top for us now but I was thinking that if someone would indeed like to have a catalogue (a type of Creative Work?), it would include multiple products potentially. Not sure how to represent this. And whether this is actually relevant for us.